components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/NewVersionNotifierConfigurations.java

```diff
@@ -0,0 +1,29 @@
+package org.wso2.carbon.apimgt.core.configuration.models;
```

Comment:
License header is missing. Please check all the java classes and add the license header.